data/json/items/tool/science.json

```diff
@@ -605,6 +612,7 @@
     "type": "TOOL",
     "name": { "str": "hydrogen tank" },
     "description": "A tank of compressed hydrogen gas.  If you need to make water from scratch, or lift a zeppelin, it could come in handy.",
+    "category": "container",
```

Comment:
It doesn't function as a container. Not sure where it'd fit best, but this is realistically either spare parts or chems. It's a glorified resource item for hydrogen.

Reply:
put hydrogen & nitrogen tank in other, No uses for them while looking through HHG.